app/javascript/profileDropdown/abuseButton.js

```diff
@@ -0,0 +1,43 @@
+export default function initAbuseButton() {
+  const abuseButton = document.getElementById(
+    'user-profile-dropdownmenu-abuse-button',
+  );
+  if (!abuseButton) {
+    // button not always present when this is called
+    return;
+  }
```

Comment:
This exists almost line for line in the blockButton.js file, so we might want to look for a way to eliminate that redundancy at some point.